src/cctz_libc.cc

```diff
@@ -20,6 +20,7 @@
 #include <ctime>
 
 namespace cctz {
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
Sorry, I missed that these were #ifndef's, so the ones in the cctz_libc files will have to be #if !(defined(_WIN32) || defined(_WIN64)).

But what's actually preventing these from working in Visual Studio? Is it just tm_gmtoff and tm_zone? I think tm_gmtoff is indirectly accessible through _isindst, _getdstbias, and _gettimezone. tm_zone is returned by _get_tzname, but that doesn't return the time zone abbreviation. It is what VS's strftime prints for %Z so cctz::Format would be consistent with the "native" strftime, if that's acceptable. Windows doesn't seem to have the ability to retrieve the abbreviation for a specific time zone.